### PR TITLE
Save branch

### PR DESCRIPTION
**Pull request by NaveenGoli**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@NaveenGoli__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ngoli). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/Common/Commands.ScenarioTests.Common/Commands.ScenarioTests.Common.csproj [hunk 1]
```diff
@@ -129,6 +129,10 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\ResourceManager\Websites\Commands.Websites\Commands.Websites.csproj">
```
Comment (markcowl) [inline src/Common/Commands.ScenarioTests.Common/Commands.ScenarioTests.Common.csproj]: This shouldn't be necessary

File: src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    </Reference>
+    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.12.111071459\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms, Version=2.11.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.12.111071459\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.WindowsAzure.Management, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (markcowl) [inline src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj]: Why do you need this?

File: src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebsitesTests.ps1 [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests creating a new website.
+#>
+function Test-CreatesNewSimpleWebsite
+{
+	# Setup
+	$rgname = "Default-Web-WestUS"
```
Comment (markcowl) [inline src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebsitesTests.ps1]: You should not use static names like this - it does not scale and makes it impossible to point a test at a different environment.  Instead, use existing functions to get the locatiosn for your RP and prefer West US'

File: src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebsitesTests.ps1 [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+<#
+.SYNOPSIS
+Tests creating a new Web Hosting Plan.
+#>
+function Test-CreatesNewWebHostingPlan
+{
+	# Setup
+	$rgname = "Default-Web-WestUS"
```
Comment (markcowl) [inline src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebsitesTests.ps1]: same comment

**Comment by markcowl** [comment]
@NaveenGoli Reviewed, please respond to feedback

**Comment by ogail** [comment]
test this please

**Comment by NaveenGoli** [comment]
Seems like this passed, what do you want me to test?

Sent from my Windows Phone
________________________________

**Comment by azurecompliance4** [comment]
That's just an instruction to the Jenkins server to run the job.
________________________________

**Pull request merged**